content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -153,6 +155,10 @@ wait until the newly defined RBAC policy to take effect.
 
 1.  Enable the Istio RBAC for the namespace:
 
+    {{< warning >}}
+    This is a cluster-wide policy and Istio overwrites any another `ClusterRbacConfig`.
```

Comment:
Please include the nouns. For example `This` without a noun might lead to readers not knowing what you are referencing. Consider:
```suggestion
    The Istio RBAC enablement is a cluster-wide policy and Istio overwrites any previously set `ClusterRbacConfig` configuration.
```